parse-package-set/src/Main.purs

```diff
@@ -49,13 +54,16 @@ main :: Effect Unit
 main = runAff_ (either throwException mempty) do
   tmpPath <- liftEffect $ tmpdir "cst-integration-"
 
-  writeTextFile UTF8 (tmpPath <> "/spago.dhall") defaultSpagoDhall
+  liftEffect $ Console.log $ "Making new project in " <> tmpPath
 
-  let execOpts = Exec.defaultExecSyncOptions { cwd = Just tmpPath }
-  s <- liftEffect $ Buffer.toString UTF8 =<< Exec.execSync "spago ls packages" execOpts
-  let lines = Str.split (Str.Pattern "\n") s
-  let packages = Str.joinWith " " (String.takeWhile (_ /= ' ') <$> lines)
-  _ <- liftEffect $ Exec.execSync ("spago install " <> packages) execOpts
+  writeTextFile UTF8 (tmpPath <> "/spago.yaml") defaultSpagoYaml
+
+  let execOptsFn execSyncOptions = execSyncOptions { cwd = Just tmpPath }
```

Comment:
Maybe just inline this as `_ { cwd = Just tmpPath }`? I think that's the intended usage of this API.

Reply:
I see you updated this to a section, but I think it should just be inlined.